examples/.eslintrc

```diff
@@ -1,22 +1,30 @@
 {
-    "extends": ["gooddata"],
+    "extends": "gooddata",
 
     "rules": {
         "class-methods-use-this": 0,
-        "max-len": [0],
-        "no-underscore-dangle": [0],
-        "mocha/no-hooks-for-single-case": 0,
-        "react/no-find-dom-node": 0,
-        "react/no-multi-comp": 0,
-        "react/no-unused-prop-types": [0, { "skipShapeProps": true }],
-        "react/no-string-refs": 0,
-        "react/forbid-prop-types": 0,
-        "jsx-a11y/img-has-alt": 0,
+        "jsx-a11y/heading-has-content": 0,
         "jsx-a11y/label-has-for": 0,
-        "jsx-a11y/no-static-element-interactions": 0
-    },
+        "jsx-a11y/no-static-element-interactions": 0,
+        "react/forbid-prop-types": 0,
+        "react/no-find-dom-node": 0,
+        "react/no-unused-prop-types": 0,
+
+        "function-paren-newline": 0,
+        "jsx-a11y/click-events-have-key-events": 0,
+        "jsx-a11y/mouse-events-have-key-events": 0,
+        "object-curly-newline": 0,
+        "prefer-destructuring": 0,
+        "prefer-promise-reject-errors": 0,
+        "react/default-props-match-prop-types": 0,
+        "react/jsx-curly-brace-presence": 0,
+        "react/no-typos": 0,
 
+        "newline-per-chained-call": 0,
```

Comment:
This configuration didnt work... :(
`"newline-per-chained-call": ["error", { "ignoreChainWithDepth": 2 }]`